ansible_collections/arista/avd/docs/templates/plugin-docs.j2

```diff
@@ -0,0 +1,163 @@
+{#
+ Copyright (c) 2023 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# Based on https://github.com/xlab-steampunk/ansible-doc-extractor/blob/master/src/ansible_doc_extractor/templates/module.md.j2 #}
+<!--
+  ~ Copyright (c) 2023 Arista Networks, Inc.
+  ~ Use of this source code is governed by the Apache License 2.0
+  ~ that can be found in the LICENSE file.
+  -->
```

Comment:
so... do we need copyright in auto-generated files? ... :) (we can approve like this in any case, it just seems overkill)

Reply:
if we go to this extent we could add a line as comment to say "this has been auto-generated, do not edit"

Reply:
It will be removed again once we get the mkdocs diff stuff sorted out.